src/styles/navbar.scss

```diff
@@ -248,6 +248,13 @@ button {
     max-height: 2rem;
   }
 
+  .navbar-item .join-slack {
+    transition: 0.5s all;
+    max-height: 100%;
+    padding: 0 0.1rem;
+    width: 100%;
```

Comment:
You can remove this line as it makes the button exceptionally large on small devices. 
```suggestion
```

Reply:
I tried these changes but while accessing the website on mobile devices it looks kinda odd since all other nav-items have 100% width.

- **without width 100% left aligned**
![Screenshot from 2021-10-02 23-57-42](https://user-images.githubusercontent.com/51537596/135728026-e58f3fc9-26c7-4259-af60-c1915b4bea41.png)

- **without width 100% center aligned**
![Screenshot from 2021-10-02 23-58-08](https://user-images.githubusercontent.com/51537596/135728041-38f7abda-ca6c-41e2-8037-24b6a6720006.png)

So, I decided to go with this styling since it was the best option.

Reply:
> without width 100% and left aligned

This looks good for now, we would be going a design change just before the event so most of things would be changing look wise. But your thinking is understandable and on point.